html/login/controllers/authentication.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require_once "../models/user.php";
+
+class AuthenticationController {
+    public function authenticate($username, $password) {
+        $userModel = new UserModel();
+        $user = $userModel->authenticateUser($username, $password);
+
+        if ($user) {
+            header("Location: ../views/dashboard.php");
+            exit();
+        } else {
+            // Redirect to the login page with an error message
```

Comment:
remove comment

Reply:
Yes sir, i already resolved and pushed it. Thank you.